### ISSUE TITLE
[BUG]: [ERROR]: Nothing was scraped

### ISSUE THREAD

**Issue by pr0k41** [pystardust/ytfzf#708]
### Describe the bug
I get the error when i search anything
### To Reproduce
Just search something
### Expected behavior
to search and show output in the form of video titles
### Screenshots
![image](https://github.com/pystardust/ytfzf/assets/114640991/35bc4fcd-7bc9-400a-b20f-c107a8017bc8)
### Information
 - OS:Exodia OS
 - Terminal:Alacritty
 - Ytfzf version:2.5.5-4
 - Output of `readlink $(which sh)`:bash
 - (if is a thumbnail issue) run `ytfzf --thumbnail-log=log.txt` and post the file:

**Comment by Euro20179** [comment]
Try using `ytfzf --rii`

**Comment by groggone** [comment]
> Try using `ytfzf --rii`
Doesn't work

**Comment by Euro20179** [comment]
Could also try running
```sh
invidious_instance="https://vid.puffyan.us" ytfzf
```

**Comment by sangram88** [comment]
it works now we are seeing errors when loading file

**Comment by parzivalmrm** [comment]
where exactly in the config file is it supposed to go.

`invidious_instance="https://vid.puffyan.us"`

**Comment by userAndri** [comment]
> where exactly in the config file is it supposed to go.
> 
> `invidious_instance="https://vid.puffyan.us"`

It goes here: sudo nano ~/.config/ytfzf/conf.sh

**Comment by rexysans** [comment]
Hey @pr0k41,

I ran into the same issue, and this worked for me:

invidious_instance="https://vid.puffyan.us" ytfzf  

If that solves it for you, you can make it permanent by editing your ~/.bashrc or ~/.zshrc file and adding:

export INVIDIOUS_INSTANCE="https://vid.puffyan.us"  

Then reload your shell with:

source ~/.bashrc  # or ~/.zshrc  

If that doesn’t work or you want a quick workaround, just run this alias:

alias ytfzf='invidious_instance="https://vid.puffyan.us" ytfzf'  

Add it to your shell config if you want it to stick around.

**Comment by tharun983** [comment]
its working but why is the video loading so slow

**Comment by rexysans** [comment]
> its working but why is the video loading so slow

Then check out **yt-x** by Benexl at https://github.com/Benexl/yt-x
It’s fast and comes with more features than ytfzfz.

**Comment by Ravendon** [comment]
I installed it in Debian. Same error and nothing here fixed it.

**Comment by plasmoduck** [comment]
Yeah, this used to be a great program. I used it for years. Now it doesn't work. Always nothing scraped. I could just use it without any arguments 'ytfzf foobar' and it would scrape youtube and show me all the videos. Now its always [ERROR]: Nothing was scraped.

**Comment by onemoresuza** [comment]
There's seems to be an alternative being developed: <https://github.com/Benexl/yt-x>.

**Comment by plasmoduck** [comment]
Yeah I've since switched to using that.

**Comment by tabletseeker** [comment]
Updated and working ytfzf project here:
https://github.com/tabletseeker/ytfzf_prime
